pyqldb/execution/executable.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
s/2019/2020 or remove completely

Reply:
fair point.
I actually want to raise a separate PR for all files.
Don't want to bloat the diff.